code/modules/capture_the_flag/ctf_game.dm

```diff
@@ -389,7 +389,7 @@
 
 	var/mob/living/carbon/human/M = new /mob/living/carbon/human(get_turf(src))
 	new_team_member.prefs.safe_transfer_prefs_to(M, is_antag = TRUE)
-	M.set_species(/datum/species/synth)
+	M.set_species(/datum/species/human)
```

Comment:
you can just get rid of this line afaik they're already human

Reply:
safe_transfer_prefs_to() calls set_species() to their pref species iirc